coverage-tests.js

```diff
@@ -1350,25 +1351,20 @@ test('should find regions by visual locator', {
 })
 
 test('should extract text from regions', {
-  page: 'StickyHeader',
+  page: 'OCR',
   config: {stitchMode: 'CSS'},
   test({driver, eyes, assert}) {
     eyes.open({appName: 'Applitools Eyes SDK', viewportSize})
-    const element = driver.findElement({type: 'css', selector: '.page h1'})
+    const element = driver.findElement({type: 'css', selector: 'body > h3'})
     const texts = eyes.extractText([
-      {target: {left: 38, top: 38, width: 213, height: 23}, hint: 'This is the navigation bar'},
+      {target: {left: 8, top: 21, width: 400, height: 37}, hint: 'Header1: Hello world!'},
       {target: element},
-      {target: '.page p:nth-of-type(3)'}
+      {target: 'body > ol > li:nth-child(1)'}
```

Comment:
I'd rather it be a paragraph. Let's use the `p` element in the page instead, and change its style to have `width:500px` so that it won't be sensitive to scrollbars and resize.

Reply:
the expected text would then be:
```
Incididunt minim ad occaecat mollit sint elit ipsum. Consectetur eiusmod sint
officia labore elit nostrud mollit eiusmod reprehenderit excepteur. Non
aliquip ipsum adipisicing ut consectetur qui ipsum et labore enim irure ex.
Voluptate voluptate sit ex consectetur nisi sunt dolor.
```

It would also make sense to make the font not a system font. You can use Roboto like in the other pages.